spec/requests/set_user_after_login_spec.rb

```diff
@@ -11,7 +11,10 @@
       get '/auth/myusa/callback'
     }.to change { User.count }.by(1)
 
-    expect(User.last.email_address).to eq('george-test@example.com')
+    new_user = User.last
+    expect(new_user.email_address).to eq('george-test@example.com')
+    expect(new_user.first_name).to eq("Georgie")
```

Comment:
I guess what I was thinking was set up something like this that does not include fname / lname and log in with that: https://github.com/18F/C2/blob/b25faa52f4c75b8d1e56127251a805ea38e055f0/spec/support/integration_spec_helper.rb